conf/defaults.ini

```diff
@@ -1170,3 +1170,22 @@ default_baselayer_config =
 
 # Enable or disable loading other base map layers
 enable_custom_baselayers = true
+
+#################################### Dashboard previews #####################################
+
+[dashboard_previews.crawler]
+# Number of dashboards rendered in parallel. Default is 6.
+thread_count =
```

Comment:
I'm noticing for some defaults we have explicitly set a value (such as `true` above for base layers). Is the behavior of this file to fallback to a hardcoded value (`6` in the case of `thread_count`) else override with value listed here? Wondering as if that's not the case why not just set defaults inline here?

Reply:
I am not sure about the convention - I followed the pattern used in `plugin.grafana-image-renderer` section.

I guess the hardcoded value in `must` is handy because it will be used even if I type `thread_count = ab` in my `custom.ini`. I would honestly prefer to just fail fast, but...